### PR TITLE
feat(home): add home page

### PR DESCRIPTION
**Pull request by mjulialobo**
## Why
add home page

## What was done
- Created banner and products display
- Unit tests
- Styled desk and mobile

## How to test
- Open the home page, see if its responssive

### CODE REVIEW

File: src/screens/Home/HomeDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  MOST_WANTED_COMPLEMENT,
+} from "./constants";
+
+export type HomeDetailsProps = {
+  products: Product[];
+};
+
+export const HomeDetails = ({ products }: HomeDetailsProps) => {
+  return (
+    <>
```
Comment (priscilacng) [inline src/screens/Home/HomeDetails.tsx]: you dont need this extra `<>` since you have a single container

File: src/screens/Home/HomeDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          </BannerTextContent>
+        </Banner>
+        <ProductDisplay>
+          <ProductDisplayTextContent>
+            <ProductDisplayTitle>{MOST_WANTED}</ProductDisplayTitle>
+            <Text>{MOST_WANTED_COMPLEMENT}</Text>
+          </ProductDisplayTextContent>
+          {products.map(
+            (product, index) =>
+              index <= 3 && <ProductCard key={index} product={product} />
```
Comment (priscilacng) [inline src/screens/Home/HomeDetails.tsx]: you could use a constant here to describe this number
or you could use `products.slice(0, 3)` to get only the ones you need

File: src/screens/Home/HomeDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          )}
+
+          <Link href="/">
+            <Image
+              src="https://cdn-icons-png.flaticon.com/128/6318/6318306.png"
+              data-testid="navigate-icon"
+              width={60}
+              height={60}
+              alt="navigate-icon"
+              className="navigate-icon"
```
Comment (priscilacng) [inline src/screens/Home/HomeDetails.tsx]: is there a reason to use `className` instead of creating a new component?

Comment (mjulialobo) [reply src/screens/Home/HomeDetails.tsx]: Actually, I needed it for the banner to add a specific style to it and use next image, but for this guy, I'll remove it!

**Pull request merged**